pkg/extensions/search/cve/testdata/zot-dis-test/blobs/sha256/5de64447fc6d0caa31843bd7dd43b2fdde02475403d1ea016a9e512e02790166

```diff
@@ -0,0 +1 @@
+{"created":"1970-01-01T00:00:00Z","author":"Bazel","architecture":"amd64","os":"linux","config":{"Env":["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin","SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt"]},"rootfs":{"type":"layers","diff_ids":["sha256:59839d326e715c1ff37d9bf6ce1186a4ab85f3dfd6fcfee9a614e7e2a136fc97","sha256:b335a3ef21dea5a9b17da691ae44b7a344e6a1fa649c7aa5ae61216ca42b1f46"]},"history":[{"created":"1970-01-01T00:00:00Z","created_by":"bazel build ...","author":"Bazel"},{"created":"1970-01-01T00:00:00Z","created_by":"bazel build ...","author":"Bazel"}]}
```

Comment:
Did we forget to remove some files? What are all these blob thingies?

Reply:
This are the test files I added to scan an image.